### PR TITLE
Add support for bulk adding reviewers

### PR DESCRIPTION
**Pull request by dpursehouse**

### CODE REVIEW

File: bulk.py [hunk 1]
```diff
@@ -36,6 +36,9 @@ def _main():
     parser.add_argument('--hashtag', dest='hashtags',
                         required=False, action='append',
                         help='add hashtags')
+    parser.add_argument('-r', '--reviewer', dest='reviewers',
+                        required=False, action='append',
+                        help='add reviwers')
```
Comment (marco-miller) [inline bulk.py]:
```suggestion
                        help='add reviewers')
```

**Comment by marco-miller** [comment]
You might prefer to squash the 2 commits together before merging.

**Comment by marco-miller** [comment]
Stumbled upon this still open PR. Was I to do anything for it? -Not sure anymore.

**Comment by dpursehouse** [comment]
Thanks, I forgot about this.  This was closed by f1677ab5f26b9f13cee6ce339c7286fba241657c in which I squashed your suggested change.  I forgot to add a reference to the PR in the commit message so that it would be automatically closed on push.

**Pull request closed without merging**